monorepo-migration/data/repo-numbers.json

```diff
@@ -30202,7 +30191,6 @@
             28142,
             28145,
             28150,
-            28152,
```

Comment:
It's weird these PRs/issues disappeared from Github. E.g.: https://github.com/mozilla-mobile/fenix/issues/28152

That said, there's nothing we can do in this current PR.